.github/stale.yml

```diff
@@ -0,0 +1,26 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```

Comment:
What about DO NOT MERGE?

Reply:
We shouldn't have them as PR's, if they are being used by docs, they should be on official branches we review.  If a DNM isn't touched for 30 days, someone should ping.